core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java

```diff
@@ -658,27 +658,27 @@ protected KIP7(String contractAddress, Caver caver, TransactionManager transacti
     }
 
     /**
-     * Static method create KIP7 Standard Token handler connecting existed Contract
+     * Static method create KIP-7 Standard Token handler connecting existed Contract
      * The credential with the "KlayCredential Object" passed as a function parameter will control the contract
      * @param contractAddress KIP7 Standard Token Address
```

Comment:
```suggestion
     * @param contractAddress KIP-7 Standard Token Address
```

Reply:
Let me know what do you think. There are more items to be changed like this. Do we need to change this? If yes, please do that for all other items in this code. If not, please ignore this comment.

Reply:
This PR was intended to change the word "KIP7" written in the function description and function argument in the comments. So, I think it's a right to consistently change all items listed in function descriptions, function arguments. 
I will check and replace it once again.

Reply:
I modified all item in function description and argument.